examples/rosettacode/factorial.nit

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env nit
+#
+# This file is part of NIT ( http://www.nitlanguage.org ).
+# This program is public domain
+
+# Task: Factorial
+# SEE: <http://rosettacode.org/wiki/Factorial>
+
+module factorial
+
+redef class Int
+
+    fun fact: Int
```

Comment:
yes, but isn't the point of the task to show how to implement it? But I agree that something like the following should be added at the end of the file.

~~~nit
# By the way, `factorial` already exists in Int class
print n.factorial
~~~